activities/organization/finance-updates.md

```diff
@@ -0,0 +1,19 @@
+---
+type: resource
+---
+
+# Finance updates
+
+For each board meeting the Operations Coordinator prepares a finance update with details on current cash position, costs incurred and the projected runway. The update also sheds a light on other financial aspects, such as fundraising results and general financial updates.
+
+## September 2020
```

Comment:
For easier future maintenance/scanability, this could become:

# Financial updates

* [Sept 2020 financial update]
* [some other date]